servers/src/main/java/tachyon/master/next/block/BlockMaster.java

```diff
@@ -69,6 +78,12 @@
    */
   private final BlockIdGenerator mBlockIdGenerator = new BlockIdGenerator();
   private final Set<Long> mLostBlocks = new HashSet<Long>();
+  private final BlockingQueue<MasterWorkerInfo> mLostWorkers =
+      new LinkedBlockingQueue<MasterWorkerInfo>();
+  private final TachyonConf mTachyonConf;
+
+  // heart beat service
+  private Future<?> mHeartbeat;
```

Comment:
This is the heartbeat for detecting lost workers, so it the comments should reflect that, and maybe a more descriptive variable name.